assignments/pig-latin/code.js

```diff
@@ -31,3 +31,17 @@
 //
 // Feel free to play around in the REPL to make sure you understand how this
 // operator works.
+
+
+
+
+
+const pigLatin = (w) => {
+  var l= (w.search(/[aeiou]/))
```

Comment:
Don't use `var`. In this case you can use `const` since the variable's value never changes. If it did need to change, you should use `let`. `var` is old fashioned and has some subtle technical problems.

Reply:
Also, `l` is not a great variable name since it's easy to confuse with a `1`. `i` would be a great name for this variable since it's an index value.

Reply:
Oh, and finally, you don't need the `()`s around the expression. So all-in-all this line should probably look like:

```suggestion
  let i = w.search(/[aeiou]/);
```